test/1_stdlib/ArrayCore.swift

```diff
@@ -21,7 +21,7 @@ import Swift
 var trackedCount = 0
 var nextTrackedSerialNumber = 0
 
-final class Tracked : ForwardIndex, CustomStringConvertible {
+final class Tracked : Equatable, CustomStringConvertible {
```

Comment:
Given the usage below:

```
((Tracked(10)..<Tracked(27)).makeIterator())
```

I think it needs to be `Strideable`.

Also, please feel free to add `Strideable` conformance to `stdlib/private/StdlibUnittest/LifetimeTracked.swift` and replace `Tracked` with `LifetimeTracked`.  This test predates the unittesting infrastructure.